src/mat73.c

```diff
@@ -3173,16 +3178,19 @@ Mat_VarReadNextInfoIterate(hid_t id, const char *name, const H5L_info_t *info, v
 
     /* FIXME: follow symlinks, datatypes? */
 
+    mat_data = (struct ReadNextIterData *)op_data;
+
     /* Check that this is not the /#refs# or /"#subsystem#" group */
     if ( 0 == strcmp(name, "#refs#") || 0 == strcmp(name, "#subsystem#") )
         return 0;
+    if (mat_data && mat_data->pred && mat_data->pred(name, mat_data->pred_user_data) == 0)  // do we need to skip it?
```

Comment:
Please use a C89 style comment.

```suggestion
    if (mat_data && mat_data->pred && mat_data->pred(name, mat_data->pred_user_data) == 0)  /* do we need to skip it? */
```